.github/workflows/macos.yml

```diff
@@ -129,12 +129,6 @@ jobs:
       - name: Build And Test
         shell: bash
         run: |
-          if [ "${{ matrix.EVENT_MATRIX }}" == "OPENSSL_1_1" ]; then
-            export PKG_CONFIG_PATH="/usr/local/opt/openssl@1.1/lib/pkgconfig:$PKG_CONFIG_PATH"
```

Comment:
Am I missing something, or after this change default openssl will be used?
But we need to check with openssl 1.0 and openssl 1.1

Reply:
On current homebrew, `openssl@1.1` is the only openssl:

```
[pierce@plo-mbp15 libevent]$ brew --prefix openssl
/usr/local/opt/openssl@1.1
[pierce@plo-mbp15 libevent]$ brew info openssl
openssl@1.1: stable 1.1.1g (bottled) [keg-only]
Cryptography and SSL/TLS Toolkit
...
```

I can't find any trace of openssl-1.0.x in homebrew anymore.

Reply:
Indeed.  OpenSSL 1.0 was removed from homebew in https://github.com/Homebrew/homebrew-core/pull/46876. Related discussion also in https://github.com/Homebrew/homebrew-core/issues/46454.

Reply:
Ok, then it is better to remove 1.0 openssl firstly with a commit message that will include the information above and after remove exports

Reply:
Refixed (cmake still there, build matrix element is also had not been removed)

Reply:
But it is presented in the github ci:

```
2020-11-01T19:13:54.2224660Z configure: PKG_CONFIG_PATH has set to /usr/local/opt/openssl@1.0.2t/lib/pkgconfig (added openssl from brew)
```

https://pipelines.actions.githubusercontent.com/GnfICO3hPviZhcl89l00Fw9DBWg9365RoSFfPB0B9kH0i2L01N/_apis/pipelines/1/runs/259/signedlogcontent/9?urlExpires=2020-11-01T19%3A40%3A50.9811884Z&urlSigningMethod=HMACV1&urlSignature=NDbTCMc5WMcU8GD3iEOOAGw7ofn4CR1wibnhQ6JROr8%3D